src/client/components/componentWithSliders/savePressetModalBlock/savePressetModalBlock.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import ModalSavePressetesWindow from './modalSavePressetesWindow';
+import BackgroundUnderModelWindow from './backgroundUnderModelWindow';
+
+class SavePressetModalBlock extends Component {
+    autoFocusModalSavePressetesWindow = (elementFromRef) => {
+      if (elementFromRef) { elementFromRef.focus(); }
```

Comment:
It’s better to not writing if statements in one line

Reply:
rewrited